java_tests/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheImpl.java

```diff
@@ -354,6 +354,72 @@ public void clear() {
         }
    }
 
+   private cli.Infinispan.HotRod.Wrappers.DictionaryOfObjects mapToDict(Map<? extends K, ? extends V> m)
+   {
+	   cli.Infinispan.HotRod.Wrappers.DictionaryOfObjects dict = new cli.Infinispan.HotRod.Wrappers.DictionaryOfObjects();
+       for (Entry<? extends K, ? extends V> entry : m.entrySet()) {
+           dict.Add(marshal(entry.getKey()), marshal(entry.getValue()));
+       }
+       return dict;
+   }
+   
+   @Override
+   public CompletableFuture<Void> putAllAsync(Map<? extends K, ? extends V> map) {
+       try {
+           if (false) workaroundCLICheckedExceptions();
```

Comment:
Uh sorry, I didn't reply to this. It's an old piece of code, I'm not sure if there's actually a workaround behind, maybe for IKVM. I hadn't time to try a build without these lines. We can leaves these as they are just for now? I'll clean up the code later.